### PR TITLE
move Model CI docs to its own page

### PR DESCRIPTION
**Pull request by drakejwong**
Autodeployed docs: https://scale-ai-nucleus-python-sdk--176.com.readthedocs.build/en/176/
* moved Model CI docs to its own page
* cleaned up some imports
    * let me know if this was breaking
    * or if you want to change the members surfaced under `nucleus.modelci`

### CODE REVIEW

File: nucleus/modelci/__init__.py [hunk 1]
```diff
@@ -1,4 +1,19 @@
...
+    "Connection",
+    "EvalFunctionEntry",
+    "EvaluationCriterion",
+    "GetEvalFunctions",
+    "ModelCI",
+    "ThresholdComparison",
+    "UnitTest",
+    "UnitTestEvaluation",
+    "UnitTestItemEvaluation",
+    "UnitTestMetric",
```
Comment (pfmark) [inline nucleus/modelci/__init__.py:14]: imo we can just have `ModelCI` and `UnitTest` here for now. The others are rather confusing for the beginning. Wdyt @sasha-scale ?

**Review by pfmark — APPROVED** [review]
Awesome, thanks @drakejwong ! Let's just keep the two classes I listed, then lgtm! Great stuff :) Everything on right on time :D

**Comment by drakejwong** [comment]
Fixed! Very easy to re-add members later.

One of the tests is red for me, maybe because I broke imports. Shall merge once it's green!

**Comment by drakejwong** [comment]
test failure is unrelated afaict so I'm merging

**Pull request merged**